src/main/app/fact/FactApi.ts

```diff
@@ -45,6 +47,34 @@ export class FactApi {
       .catch(this.errorHandler({}));
   }
 
+  public getOpeningTimeTypes(): Promise<OpeningType[]> {
+    return this.axios
+      .get(`${this.adminBaseUrl}/openingTypes`)
+      .then(results => results.data)
+      .catch(err => {
+        this.errorHandler([]);
+        return Promise.reject(err);
```

Comment:
Just to double check on this one, the error will be raised back to the calling function on the controller, and from there the error message will be displayed. 

Will error be displayed to the user? And if not, will it be a general error that we have defined? If so, then great! If not, then we may want to log it in a debug sense to help us in the future

Reply:
The call to the existing error handler logs the error. Returning the rejected promise allows us to catch this error when calling the API method. In the case of opening times, we don't display the specific error returned in the UI but we do catch the error and display a meaningful message, such as: 'A problem occurred when retrieving the opening times.'